source/sass/components/image-text.scss

```diff
@@ -0,0 +1,23 @@
+.div {
```

Comment:
As second note: does this need additional namespacing, or is this supposed to apply to the entire website? (since main.scss just imports the file without any namespace wrapping, in theory everything could be affected by these classes)

Reply:
@Pomax not sure I understand the second comment. Do you mind clarifying what you mean by namespacing/wrapping, etc.?

Reply:
Are you suggesting that I rename the classes?...If so, I can definitely do that.

btw, these classes are only meant to affect this component, not the entire site.

Reply:
right, so right now these classes kick in at the site level, rather than being namespaced inside, say `.cms .image-text { ... }` or something else that makes sure they'll only kick in for this component rather than "for everything". So can we make sure that these rules are wrapped by a CSS selector that keeps things local to the component?

Reply:
okay, good idea. if that's the case, I need to go back and fix this on #2722 as well. i'll make a separate ticket and fix that one as well.